cl/lib/pacer_session.py

```diff
@@ -79,3 +82,27 @@ def get_pacer_cookie_from_cache(user_pk: Union[str, int], r: Redis = None):
     pickled_cookie = r.get(session_key % user_pk)
     if pickled_cookie:
         return pickle.loads(pickled_cookie)
+
+
+def refresh_pacer_cookies(
+    user_pk: Union[str, int],
+    username: str,
+    password: str,
+    client_code: str = None,
+) -> RequestsCookieJar:
+    """Refresh PACER cookies for a user.
+
+    :param user_pk: The PK of the user attempting to store their credentials.
+    Needed to create the key in Redis.
+    :param username: The PACER username of the user
+    :param password: The PACER password of the user
+    :param client_code: The PACER client code of the user
+    :return: Cookies for the PACER user
+    """
+
+    r = make_redis_interface("CACHE", decode_responses=False)
+    cookies = log_into_pacer(username, password, client_code)
+    cookie_expiration = 60 * 60
+
+    r.set(session_key % user_pk, pickle.dumps(cookies), ex=cookie_expiration)
+    return cookies
```

Comment:
This violates DRY. Can you factor it out along with the code above into a new function?

Reply:
Done, I've merge this function into `get_or_cache_pacer_cookies`